README.md

```diff
@@ -18,17 +40,9 @@ We also support passing a list of strings that make up the bundle by specifying
 
 `api.build_verify_and_push($your_namespace, $your_repository, $your_release_version, $your_quay_authtoken, yamls=$your_yamls_list)`
 
-Not only that, but we also support the use of the tool as a command line interface. Just invoke `operator-courier` from a shell:
-
-```bash
-$ operator-courier verify --source_dir="./temp/bundles/couchbase/"
```

Comment:
Do we want to omit mention of the verify command? I agree that the focus should be on the push command, but this entirely removes mention of it.

Reply:
That's a good point. It makes sens to call out how to verify it too.

Reply:
included in latest push.